tests/unit/base/trained_model_serialization_test.cc.py

```diff
@@ -35,6 +35,17 @@
     'CMulticlassLibSVM' # error C <= 0
 ]
 
+# Algorithms that can be prematurely stopped
+STOPPABLE = [
```

Comment:
I have another Q here: rather than whitelisting stoppable machines, we should backlist the ones that we want to exclude from testing. Or? Otherwise new machines will not be added to this list, and thus, not be tested

Reply:
i think its better to whitelist because blacklisting will mean having to blacklist the same algorithms over again for each test.. for example: we will have to add the algos in the ```IGNORE``` list here as well (since they cannot be initialized in the same way)

Reply:
sure there will be different blacklists for different tests, but that is fine. Some machines will be serializable but not stoppable and vice versa. As said before, if we have to explicitly add new machines to this list in order to have them tested, this will be forgotten and so test coverage will suffer. Think 3 years ahead. We can maybe have a global ignore list and a test specific one to avoid too much redundancy

Reply:
you are right i did miss that....